clients/fides-js/src/lib/tcf/utils.ts

```diff
@@ -0,0 +1,64 @@
+import { TCModel, TCString, Vector } from "@iabtechlabtcf/core";
+import { PrivacyExperience } from "../consent-types";
+import { EnabledIds, TcfCookieConsent, TcfCookieKeyConsent } from "./types";
+import { TCF_KEY_MAP } from "./constants";
+import { generateFidesString } from "../tcf";
+import { debugLog } from "../consent-utils";
+
+export const transformFidesStringToCookieKeys = (
+  fidesString: string,
+  debug: boolean
+): TcfCookieConsent => {
+  // Defer: to fully support AC string, we need to split out TC from AC string https://github.com/ethyca/fides/issues/4265
```

Comment:
@allisonking I went ahead and updated more usages of `tcString` to `fidesString` to complete most of the work needed here- https://github.com/ethyca/fides/issues/4265, but there's still the question of splitting out the TC portion from the `fidesString`, along with adding tests and some validation checks to fully support AC str here. 

Do we have a separate ticket for functionality that's separate from just naming?

Reply:
I think that can be this one? https://github.com/ethyca/fides/issues/4263 can you add a comment to that ticket if you think there's stuff not captured there?

Reply:
oh perfect, thanks!